src/fhir_search/uri_parser.clj

```diff
@@ -0,0 +1,70 @@
+(ns fhir-search.uri-parser 
```

Comment:
lets talk about the subject instead of the verb, I mean, instead of `parser` I if we name it `query`? Actually this ns will be responsible for handling URL queries. After all, java comes with a built-in URL/URI parsing facility, what is different in ours is that we are dealing with a fhir compliant URL query.

`fhir-search.url-query`